app/uk/gov/hmrc/helptosavefrontend/connectors/EmailVerificationConnector.scala

```diff
@@ -43,34 +43,33 @@ trait EmailVerificationConnector {
 @Singleton
 class EmailVerificationConnectorImpl @Inject()(http: WSHttp, conf: Configuration) extends EmailVerificationConnector with ServicesConfig with Logging {
 
-  val linkTTLMinutes = conf.underlying.getInt("services.email-verification.linkTTLMinutes")
+  val linkTTLMinutes = conf.underlying.getInt("microservice.services.email-verification.linkTTLMinutes")
   val emailVerifyBaseURL = baseUrl("email-verification")
   val verifyEmailURL = s"$emailVerifyBaseURL/email-verification/verification-requests"
 
   def isVerifiedURL(email: String) = s"$emailVerifyBaseURL/email-verification/verified-email-addresses/$email"
-
-  val continueURL = "email-verification.continue.baseUrl"
-  val templateId = "hts_email_verification"
+  val continueURL =  conf.underlying.getString("microservice.services.email-verification.continue-url")
+  val templateId = "awrs_email_verification"
 
   def verifyEmail(nino: String, newEmail: String)(implicit hc: HeaderCarrier): Future[Either[VerifyEmailError, Unit]] = {
-    val verificationRequest = EmailVerificationRequest(newEmail, nino, templateId, Duration.ofMinutes(linkTTLMinutes).toString, continueURL, Map())
-
+    val continueUrlWithParams = continueURL + "?p=" + EmailVerificationParams(nino, newEmail).encode()
+    val verificationRequest = EmailVerificationRequest(newEmail, nino, templateId, Duration.ofMinutes(linkTTLMinutes).toString, continueUrlWithParams, Map())
     http.post(verifyEmailURL, verificationRequest).map { (response: HttpResponse) ⇒
       response.status match {
         case OK | CREATED =>
           logger.info("[EmailVerification] - Email verification successfully triggered")
           Right(())
         case BAD_REQUEST ⇒
           logger.warn("[EmailVerification] - Bad Request from email verification service")
-          Left(RequestNotValidError(nino))
+          Left(RequestNotValidError())
         case CONFLICT ⇒
-          logger.info("[EmailVerification] - Email already verified")
-          Left(AlreadyVerified(nino, newEmail))
+          logger.info(s"[EmailVerification] - Email: $newEmail already verified")
```

Comment:
Tbh no we don't as the new email address either going to be saved in the email collection or it isn't, theres no need to log unsaved email addresses as the user will be notified of the reason and we will be logging the nino in the backend for all unsuccessful cases so we can follow unhappy user journeys in Kibana and figure out whats going wrong for them.